owncloudApp/src/main/res/xml/managed_configurations.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<restrictions xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <restriction
+        android:key="server_url_configuration"
+        android:restrictionType="string"
+        android:title="@string/server_url_configuration_title"
+        android:description="@string/server_url_configuration_description"
+        android:defaultValue="demo.owncloud.com" />
+    <restriction
+        android:key="server_url_input_visibility_configuration"
+        android:restrictionType="bool"
+        android:title="@string/server_url_input_visibility_configuration_title"
+        android:description="@string/server_url_input_visibility_configuration_description"
```

Comment:
What happens when we upload these restrictions to a MDM? Does it recognize `@string/server_url_input_visibility_configuration_description`?

Reply:
It depends on the platform. For example, in IBM MaaS360 you have to upload the `managed_configurations.xml` file by itself, so that it doesn't recognise the value of the strings. However, in Blackberry platform, the file is taken directly from the APK you upload, so they're recognised. Anyway, you're not allowed to build the app if you hardcode the strings, it results in an error of the type:

`AAPT: error: 'Hardcoded description' is incompatible with attribute description (attr) reference.`